src/components/Checkbox/Checkbox.tsx

```diff
@@ -26,53 +26,67 @@ export interface CheckboxProps extends IntrinsicElements {
  * @param {string}  [tileDescription] Text that can be used to describe the label in more detail. Activates the tile variation automatically.
  * @param {string}  [value]           Value of the input element.
  */
-export const Checkbox: React.FC<CheckboxProps> = ({
-  checked = false,
-  children,
-  disabled = false,
-  id,
-  isTile = false,
-  label,
-  name,
-  tileDescription,
-  value,
-  ...rest
-}) => {
-  const [isChecked, setChecked] = useState(checked);
-  const handleChange = () => {
-    setChecked(!isChecked);
-  };
 
-  return (
-    <div className="usa-checkbox">
-      <input
-        checked={isChecked}
-        className={
-          isTile
-            ? "usa-checkbox__input usa-checkbox__input--tile"
-            : "usa-checkbox__input"
-        }
-        disabled={disabled}
-        id={id}
-        onChange={handleChange}
-        type="checkbox"
-        name={name}
-        value={value}
-        {...rest}
-      />
-      <label className="usa-checkbox__label" htmlFor={id}>
-        {label}
-        {isTile && tileDescription && (
-          <span className="usa-checkbox__label-description">
-            {tileDescription}
-          </span>
+export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
```

Comment:
Love it! Forward them refs

Reply:
Re: demo app: exactly! In fact, I'm adding a ticket to cover that today. I have a very rough demo app I've been using to test these along the way, and I'm going to shape it into something that can also be used as good documentation. Great thinking! 🧠